shared/config/curriculum-layout.ts

```diff
@@ -0,0 +1,30 @@
+import { challengeTypes } from './challenge-types';
+import { SuperBlocks } from './superblocks';
+
+// Show a grid layout on the superblock level
+
+export const isGridBased = [
+  SuperBlocks.RespWebDesignNew,
+  SuperBlocks.JsAlgoDataStructNew,
+  SuperBlocks.SciCompPy
+];
```

Comment:
Since the client never uses this as an array, it should just be a function.

i.e. rather than having to do

```js
isGridBased.includes(aChallengeType)
```

it easier and clearer if I can write

```
isGridBased(aChallengeType)
```

```suggestion
const gridBasedSuperBlocks = [
  SuperBlocks.RespWebDesignNew,
  SuperBlocks.JsAlgoDataStructNew,
  SuperBlocks.SciCompPy
];

export const isGridBased = (superBlock: SuperBlocks) =>
  gridBasedSuperBlocks.includes(superBlock);
```